README.md

```diff
@@ -18,9 +18,12 @@ This action can be used to deploy Azure Resource Manager templates at different
 * `region`: **Conditional** Provide the target region, only required for Management Group or Subscription deployments.
 * `template`: **Required** Specify the path or URL to the Azure Resource Manager template.
 * `parameters`: Specify the path or URL to the Azure Resource Manager deployment parameter values file (local / remote) and/or specify local overrides.  
-* `deploymentMode`: `Incremental`(default) (only add resources to resource group) or `Complete` (remove extra resources from resource group) or `Validate` (only validates the template). 
+* `deploymentMode`: `Incremental`(default) (only add resources to resource group) or `Complete` (remove extra resources from resource group) or `Validate` (only validates the template) or `what-if`(to preview changes). 
```

Comment:
`what-if` is not a mode in the deployment sense afaik. We can do `what-if` for `Incremental` mode, as well as `Complete` mode